### DISCUSSION TITLE
Lab_Composition.type - будем ли профилировать?

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#258] [Лабораторная РГ]
Lab_Composition.type имеет привязку preferred к [FHIR document type codes](http://hl7.org/fhir/R4B/valueset-doc-typecodes.html). Эти коды вряд ли будут распознаны системами при обмене внутри РФ. 
Возможно, следует использовать [виды СЭМД НСИ МЗ РФ](https://nsi.rosminzdrav.ru/dictionaries/1.2.643.5.1.13.13.99.2.1079/passport/1.27), или виды учетных медицинских документов по приказам МЗ РФ.

Вопрос: будем ли профилировать type, в том числе на уровне Core и Lab?

**Comment by Vrodvi** [comment]
В этом месте виды СЭМДов не подходят, т.к. это кодирование просто "типа документа", а СЭМД - это один из форматов реализации типа документа, который может быть и в других форматах тоже реализован (к примеру, в формате PDF).

Более подходящим будет справочник типов документов - https://nsi.rosminzdrav.ru/dictionaries/1.2.643.5.1.13.13.11.1522/passport/7.24 "Виды медицинской документации"
